### DISCUSSION TITLE
Evmos Mainnet Chain Not Supported

### DISCUSSION THREAD

**Discussion by derekbar90** [WalletConnect/walletconnect-monorepo#3270] [Developer Support]
It seems that EVMOS chain is not supported on V2, the WalletConnect RPC is providing a 400 error with chain "eip155:9001" is not supported.

**Comment by derekbar90** [comment]
What is the best way to resolve this issue?

**Reply by crypblizz8** [reply]
![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/45455218/f076deec-a0f4-4575-926d-97222be831a2)
These are the EVMOS chains we support so it might be config issue. 

How are you preparing your code @derekbar90

**Reply by derekbar90** [reply]
We are currently using the useDapp library. This is the current response we are getting from the RPC: 

Response
```
{
    "status": "FAILED",
    "reasons": [
        {
            "field": "chainId",
            "description": "We don't support the chainId you provided: eip155:9001"
        }
    ]
}
```

Rrequest
```
{"id":"1690382094141913856","jsonrpc":"2.0","method":"eth_estimateGas","params":[{"from":"0x6e2c8fa48dab2db4b8dc77e015b9338c3c426b2d","to":"0x2c68d1d6ab986ff4640b51e1f14c716a076e44c4","data":"0x095ea7b30000000000000000000000001afd31627170607657224ed4ae701470209c4b2e000000000000000000000000000000000000000000000000016345785d8a0000"}]}
```

URL: https://rpc.walletconnect.com/v1/?chainId=eip155:9001&projectId=########
